### PR TITLE
feat: adding Dockerfile.chromium

### PR DESCRIPTION
**Pull request by hugo-gomes**
https://github.com/hivesolutions/headless/issues/13#issuecomment-1969911233

### CODE REVIEW

**Comment by hugo-gomes** [comment]
@joamag https://hub.docker.com/r/hivesolutions/headless/tags?page=1&name=headless-chromium

File: .github/workflows/dockerx.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Build and Push Headless Images
+'on':
```
Comment (joamag) [inline .github/workflows/dockerx.yml:2]: no need for the '

File: .github/workflows/dockerx.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  schedule:
+    - cron: 0 1 * * *
+  push:
+    branches-ignore:
+      - none
+  workflow_dispatch: null
+jobs:
+  build:
+    runs-on: self-hosted
```
Comment (joamag) [inline .github/workflows/dockerx.yml:11]: I think we can run these directly on github without s6.hive

This is an open source project

**Review by joamag — CHANGES_REQUESTED** [review]

**Comment by hugo-gomes** [comment]
@joamag https://hub.docker.com/r/hivesolutions/headless/tags?page=1&name=chromium

**Review by joamag — APPROVED** [review]

**Pull request merged**